CHANGELOG.md

```diff
@@ -172,6 +172,9 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Fix a bug on handling an invalid array value for point type field #4900([#4900](https://github.com/opensearch-project/OpenSearch/pull/4900))
 - [BUG]: Allow decommission to support delay timeout ([#4930](https://github.com/opensearch-project/OpenSearch/pull/4930))
 - Fix failing test: VerifyVersionConstantsIT ([#4946](https://github.com/opensearch-project/OpenSearch/pull/4946))
+- [BUG]: Allow decommission to support delay timeout [#4930](https://github.com/opensearch-project/OpenSearch/pull/4930))
```

Comment:
Extra line?

Reply:
:+1 fixing

Reply:
Sorry, "extra line" referred to this "Allow decommission..." line which shouldn't come in as a part of this PR

Reply:
Oh right, sorry about that, removed :)